### PR TITLE
Update manager-settings.md

### PR DESCRIPTION
**Pull request by brendankarl**
Some text, didn't have the correct setting for indent size on spaces so appeared squashed - have updated.

### CODE REVIEW

File: Viva/insights/advanced/setup-maint/manager-settings.md [hunk 1]
```diff
@@ -32,11 +32,11 @@ The size of the team counts the manager and all the employees who directly or in
 
 ## To configure manager settings
 
-Insights admins can access **Manager settings**. Before managers can access Viva Insights in Teams or on the web, or in the advanced insights app, they need to be assigned a license and have a team that meets or exceeds the **Minimum team size** setting.
+Insights admins can access **Manager settings**. Before managers can access Viva Insights in Teams or on the web, or in the advanced insights app, they need to be assigned a license and have a team that meets or exceeds the **Minimum team size** setting.
```
Comment (JohanFreelancer9) [inline Viva/insights/advanced/setup-maint/manager-settings.md:35]:
```suggestion
Insights admins can access **Manager settings**. Before managers can access Viva Insights in Teams or on the web, or in the advanced insights app, they need to be assigned a license and have a team that meets or exceeds the **Minimum team size** setting.
```

File: Viva/insights/advanced/setup-maint/manager-settings.md [hunk 2]
```diff
@@ -32,11 +32,11 @@ The size of the team counts the manager and all the employees who directly or in
...
 
 :::image type="content" source="../images/manager-settings.png" alt-text="Screenshot of Manager settings page." lightbox="../images/manager-settings.png":::
 
-1. In **Manager settings**, check the box to allow managers with the minimum team size access to their team data in Viva Insights.
+1. In **Manager settings**, check the box to allow managers with the minimum team size access to their team data in Viva Insights.
```
Comment (JohanFreelancer9) [inline Viva/insights/advanced/setup-maint/manager-settings.md:39]:
```suggestion
1. In **Manager settings**, check the box to allow managers with the minimum team size access to their team data in Viva Insights.
```

File: Viva/insights/advanced/setup-maint/manager-settings.md [hunk 3]
```diff
@@ -32,11 +32,11 @@ The size of the team counts the manager and all the employees who directly or in
...
 ![Screenshot that shows the Manager settings page.](../images/manager-settings-check-box.png)
 1. Select either **All managers**, which allows all measured managers access, or **Select managers (upload .csv)**, which enables you to give specific managers access.
```
Comment (JohanFreelancer9) [inline Viva/insights/advanced/setup-maint/manager-settings.md:41]:
```suggestion
1. Select either **All managers**, which allows all measured managers access, or **Select managers (upload .csv)**, which enables you to give specific managers access.
```

File: Viva/insights/advanced/setup-maint/manager-settings.md [hunk 4]
```diff
@@ -52,4 +52,4 @@ Insights admins can access **Manager settings**. Before managers can access Vi
 4. Select **Save**.
```
Comment (JohanFreelancer9) [inline Viva/insights/advanced/setup-maint/manager-settings.md:52]:
```suggestion
4. Select  *Save**.
```

File: Viva/insights/advanced/setup-maint/manager-settings.md [hunk 5]
```diff
@@ -20,9 +20,9 @@ audience: Admin
 >[!Important]
 > Manager settings only control **Organization trends** in the Viva Insights app in Teams and on the web. To set up or disable the **Organization trends** page, refer to [Admin tasks](/viva/insights/personal/teams/viva-teams-app-admin-tasks).
```
Comment (JohanFreelancer9) [inline Viva/insights/advanced/setup-maint/manager-settings.md:21]:
```suggestion
> Manager settings only control **Organization trends** in the Viva Insights app in Teams and on the web. To set up or disable the **Organization trends** page, refer to [Admin tasks](/viva/insights/personal/teams/viva-teams-app-admin-tasks).
```

File: Viva/insights/advanced/setup-maint/manager-settings.md (hunk 3 above)
Comment (JohanFreelancer9) [inline Viva/insights/advanced/setup-maint/manager-settings.md:41]:
```suggestion
1. Select either **All managers**, which allows all measured managers access, or **Select managers (upload .csv)**, which enables you to give specific managers access.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @brendankarl, kindly commit the suggested changes. The spacing in lines 29, 46, and 48 also needs to be corrected. Unfortunately I cannot do changes on those lines. Many thanks.

**Comment by brendankarl** [comment]
I've made the commits @JohanFreelancer9 .

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@lilyolason 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request closed without merging**